docs/pages/zenroom-javascript2b.md

```diff
@@ -220,8 +220,24 @@ Let's add also the decrypt function and we are ready to deploy our static and su
           rel.classList.add("is-success");
         }).catch(({logs}) => {
           const rel = document.getElementById('decrypted')
-          rel.value = logs
-          rel.classList.add("is-danger");  
+          // extract the error part of the logs
+          const err = JSON.parse(logs).reduce((x, l) => {
+            if (l.startsWith('J64 TRACE:')) {
+              const b64Trace = l.slice('J64 TRACE: '.length);
+              const arrayTrace = JSON.parse(atob(b64Trace));
+              for (const t of arrayTrace) {
+                if (t.startsWith('[!]')) {
+                  x.push(t);
+                }
+              }
+            } else if (l.startsWith('[!]')) {
+              x.push(l);
+              return x;
+            }
+            return x;
+          }, []);
+          rel.value = err.join("\n");
+          rel.classList.add("is-danger");
```

Comment:
uff this should really go to the `javascript/bindings` js wrapper library... how do you see her (come la vedi)?

Reply:
Maybe just a utility API? Like `getErrorFromLogs`? That extract all the strings that start with a `[!]` from the trace and the b64 trace

I would not do it by default since the other error parts are useful in case we want to debug an error

Reply:
yes something like that, `decode_error` to be consistent with the underscore naming, ideally consumed like this:

```javascript
import {zencode_exec, decode_error} from 'zenroom'


zencode_exec(scripts, {}).then(decode_error);
// same as
const result = await zencode_exec(script, {})
decode_error(result)
```

that does the decode and print them in console.log/error as the normal output <3